frontend/src/tests/pages/PlayPage.test.js

```diff
@@ -97,29 +100,65 @@ describe("PlayPage tests", () => {
         expect(mockToast).toBeCalledWith("Cow sold!");
     });
 
-    test("Make sure that both the Announcements and Welcome Farmer components show up", async () => {
+    test("3 Card components in page show up", async () => {
         render(
             <QueryClientProvider client={queryClient}>
                 <MemoryRouter>
-                    <PlayPage />
+                    <PlayPage/>
                 </MemoryRouter>
             </QueryClientProvider>
         );
-
         expect(await screen.findByText(/Announcements/)).toBeInTheDocument();
         expect(await screen.findByTestId("CommonsPlay")).toBeInTheDocument();
+        expect(await screen.findByText("Manage Cows")).toBeInTheDocument();
+        expect(await screen.findByText("Your Farm Stats")).toBeInTheDocument();
+        expect(await screen.findByText("Profits")).toBeInTheDocument();
     });
 
-    test("Make sure div has correct attributes", async () => {
+    test('should fetch data when mounted', async () => {
+        axiosMock.onGet('/api/profits/paged/commonsid?commonsId=1&pageNumber=0&pageSize=7').reply(200, pagedprofitsFixture.Page0);
         render(
             <QueryClientProvider client={queryClient}>
                 <MemoryRouter>
-                    <PlayPage />
+                    <PlayPage/>
                 </MemoryRouter>
             </QueryClientProvider>
         );
 
-        var div = screen.getByTestId("playpage-div");
-        expect(div).toHaveAttribute("style", expect.stringContaining("background-size: cover; background-image: url(PlayPageBackground.png);"));
+        const testId = "ProfitsTable";
+        await screen.findByTestId(`${testId}-cell-row-0-col-Profit`);
+        expect(screen.getByTestId(`${testId}-cell-row-0-col-Profit`)).toHaveTextContent("$12.650");
```

Comment:
Why do we want three decimal places after the decimal point?  For US Dollars we typically round to the nearest cent.

Reply:
As one step in SI prefixes is 10^3, in order to keep more information, I choose to keep 3 decimal places. However, it seems it doesn't make sense when there's no SI prefixes applied. I guess I need to add additional handling that only keeps two decimal places when no SI prefixes applied.

Reply:
It's fine for the actual numbers to have larger precision. But it doesn't make sense to show it in the UI.

Reply:
I think the users would prefer not to have the SI prefix logic at all.    It's overkill and unnecessary. Please remove it.